gradle/plugins/common/src/main/kotlin/junitbuild.java-library-conventions.gradle.kts

```
@@ -280,6 +286,9 @@ tasks {
 	checkstyleTest {
 		config = resources.text.fromFile(checkstyle.configDirectory.file("checkstyleTest.xml"))
 	}
+	rewriteDryRun {
+		dependsOn("check")
+	}
```

This is the wrong way around, isn't it?

```suggestion
	check {
		dependsOn("rewriteDryRun")
	}
```

---

yes sorry just try to execute it somehow as the build is not working on my system. 

also discovered strange behavior as build -x test does not check somehow.